packages/block-library/src/accordion/edit.js

```diff
@@ -39,8 +46,21 @@ export default function Edit( {
 		templateInsertUpdatesSelection: true,
 	} );
 
+	const addAccordionContentBlock = () => {
+		const newAccordionContent = createBlock( ACCORDION_BLOCK_NAME );
+		insertBlock( newAccordionContent, undefined, clientId );
+	};
+
 	return (
 		<>
+			<BlockControls group="other">
+				<ToolbarButton
+					label={ __( 'Add accordion content block' ) }
+					onClick={ addAccordionContentBlock }
+				>
+					{ __( 'Add' ) }
```

Comment:
I might be mistaken, but I thought we shouldn't use different labels when a button has an actual text label, from an a11y perspective.

cc @joedolson, @t-hamano

Reply:
> we shouldn't use different labels when a button has an actual text label, from an a11y perspective.

This is how I understand it too. I think we should remove the `label` prop (`aria-label`),

Reply:
Or we could do that as part of https://github.com/WordPress/gutenberg/issues/71739